gips/data/daymet/daymet.py

```diff
@@ -133,24 +139,35 @@ def __init__(self, filename):
 
 
     @classmethod
+    def query_provider(cls, asset, tile, date):
+        """Determine availability of data for the given (asset, tile, date).
+
+        Returns (basename, url) on success; (None, None) otherewise.
+        Something of a no-op for daymet as the data is daily for the
+        entire run of the dataset and the URLs are deterministic.
+        """
+        source = cls._assets[asset]['source']
+        url = (cls._assets[asset]['url'] + '/%s') % (date.year, tile, date.year, source)
+        bn = cls._basename_pat.format(asset, tile, date.year, date.strftime('%j'))
+        return (bn, url)
```

Comment:
whether double-`None` or list, it should not return info for non-fetchable data:
```
In [10]: daymetAsset.query_provider('tmin', '9404', date(1979, 5, 25))
Out[10]: 
('daymet_tmin_9404_1972146.tif',
 'https://thredds.daac.ornl.gov/thredds/dodsC/ornldaac/1328/tiles/1979/9404_1979/tmin.nc')
```
bounds checking on `cls.start_date(asset)` and `cls.end_date(asset)` is in order.